1-org/envs/shared/versions.tf

```diff
@@ -17,3 +17,16 @@
 terraform {
   required_version = ">= 0.13"
 }
+terraform {
+  required_version = ">=0.13, <0.14"
```

Comment:
there seems to be conflicting `terraform` blocks with required_version. We should keep these to `required_version >= 0.13` to allow users flexibility

Reply:
this seems to be in other updated versions files as well